src/java.net.http/share/classes/jdk/internal/net/http/ConnectionPool.java

```diff
@@ -45,14 +45,14 @@
 import jdk.internal.net.http.common.FlowTube;
 import jdk.internal.net.http.common.Logger;
 import jdk.internal.net.http.common.Utils;
+import static jdk.internal.net.http.HttpClientImpl.KEEP_ALIVE_TIMEOUT;
 
 /**
  * Http 1.1 connection pool.
  */
 final class ConnectionPool {
 
-    static final long KEEP_ALIVE = Utils.getIntegerNetProperty(
-            "jdk.httpclient.keepalive.timeout", 1200); // seconds
+    static final long KEEP_ALIVE = KEEP_ALIVE_TIMEOUT; // seconds
```

Comment:
I'd suggest removing line 55 and change the references below to KEEP_ALIVE_TIMEOUT

Reply:
Yeah fair enough actually, I neglected to notice the slight duplication of code here (in terms of readability). I'll change this and see how it looks